lib/reports/utils.ts

```diff
@@ -3,8 +3,9 @@ import nodemailer from 'nodemailer'; // Nodemailer
 /**
  * Send email to user to confirm appliction was received
  * @param to Receiver's email
+ * @param firstName Receiver's first name
  */
-const sendConfirmationEmail = (to: string): Promise<void> => {
+const sendConfirmationEmail = (to: string, firstName?: string): Promise<void> => {
```

Comment:
Why not make the `firstName` parameter non-nullable? Applications must have first names.

Reply:
The issue starts in the `payment-received` file where TypeScript was yelling at me saying that `application` could be null which made me use the chain operator `?` which makes `firstName` nullable

<img width="553" alt="image" src="https://user-images.githubusercontent.com/51224641/155457843-97786970-78c3-404c-9c23-7797a551dc88.png">


I guess `application` can't be null in this context so I could use the non-null assertion Operator `const applicationFirstName = application!.firstName;` but I'd have to ignore the eslint rule

<img width="849" alt="image" src="https://user-images.githubusercontent.com/51224641/155458032-64e25403-4554-4e04-bf32-2561fcebb10a.png">

Lmk what you think!

Reply:
I see. This is an error in our code. Any `prisma.application.findUnique` call could return `null` (if the application ID doesn't exist). We should not have used optional chaining here - instead, we should've checked if `application` is `null`, and if it is, we should return a 400 (bad request).

Could you implement that in this PR? After that is fixed, we will be guaranteed that `application` is non-null when we try to access its properties.

Reply:
Makes sense! Addressed!